### PR TITLE
Update the 'keymaps' documentation.

### PR DESCRIPTION
**Pull request by vince-fugnitto**
#### What it does

Update the documentation of the `@theia/keymaps` extension as part of the ongoing effort to improve documentation for the milestone 1.0.0 release.

Note: (the following `tslint` checks were added to the `errors.tslint.json` during testing)

```json
"rules": {
    "completed-docs": [true, "enums", "enum-members", "functions", "interfaces", "methods"],
    "no-redundant-jsdoc": true,
}
```

#### How to test

Verify the documentation.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: Vincent Fugnitto <vincent.fugnitto@ericsson.com>

### CODE REVIEW

File: packages/keymaps/src/browser/keybindings-widget.tsx [hunk 1]
```diff
@@ -94,6 +97,9 @@ export class KeybindingWidget extends ReactWidget {
      * The regular expression used to extract values between fuzzy results.
      */
     protected readonly regexp = /<match>(.*?)<\/match>/g;
+    /**
+     * The keybinding separator.
```
Comment (elaihau) [inline packages/keymaps/src/browser/keybindings-widget.tsx]: the "regex" of keybinding separator ?

File: packages/keymaps/src/browser/keybindings-widget.tsx [hunk 2]
```diff
@@ -472,8 +484,8 @@ export class KeybindingWidget extends ReactWidget {
 
     /**
      * Compare two strings.
-     * @param a {string | undefined} the first string.
-     * @param b {string | undefined} the second string.
+     * @param a the optional first string.
+     * @param b the optional second string.
```
Comment (elaihau) [inline packages/keymaps/src/browser/keybindings-widget.tsx]: would be nice if the "return" is also documented

Comment (vince-fugnitto) [reply packages/keymaps/src/browser/keybindings-widget.tsx]: Thank you! I've updated the documentation to be more thorough as suggested.

File: packages/keymaps/src/browser/keybindings-widget.tsx (hunk 1 above)
Comment (vince-fugnitto) [reply packages/keymaps/src/browser/keybindings-widget.tsx]: Thanks, I've updated the documentation to better reflect the code.

**Comment by vince-fugnitto** [comment]
Thank you for the review @elaihau !

**Pull request merged**